src/scmdata/netcdf.py

```diff
@@ -45,7 +50,13 @@ def _get_xr_dataset_to_write(run, dimensions, extras):
     return xr_ds
 
 
-def _write_nc(fname, run, dimensions, extras, **kwargs):
+def _write_nc(
+    fname: FilePath,
+    run: "BaseScmRun",
```

Comment:
`from __future__ import annotations` should remove the need for any quoting (also fine to push out into an issue if we don't want to fix here)